debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/BlockingSnapshotIT.java

```diff
@@ -146,4 +149,20 @@ private TableId tableNameId(String table) {
         return TableId.parse(DATABASE.qualifiedTableName(table));
     }
 
+    @Override
+    protected int expectedDdlsCount() {
+        return 12;
+    }
+
+    @Override
+    protected void assertDdl(List<String> schemaChangesDdls) {
+
+        assertThat(schemaChangesDdls.get(schemaChangesDdls.size() - 2)).isEqualTo("DROP TABLE IF EXISTS `blocking_snapshot_test_1`.`b`");
```

Comment:
Currently this will fail because for the `DROP` events we don't have information about the table. See https://github.com/debezium/debezium/pull/4824/files#diff-63840528f25c3335e78d8b91c2ea67602750af012c1620d9981f11cb476b480aR642